src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -198,6 +198,12 @@ ManifestCache::configManifest (
         Throw<std::runtime_error> ("Unverifiable manifest in config");
     }
 
+    // Trust our own master public key
+    if (!trusted(m.masterKey) && !unl.trusted (m.masterKey))
+    {
+        addTrustedKey (m.masterKey, "");
+    }
```

Comment:
I don't have a good answer to your question.
My change was simply to automatically trust our own manifest, instead of throwing below if the master public key is not listed in the validators file.